bayesian.go

```diff
@@ -340,6 +449,10 @@ func (c *Classifier) ProbScores(doc []string) (scores []float64, inx int, strict
 // Underflow detection is more costly because it also
 // has to make additional log score calculations.
 func (c *Classifier) SafeProbScores(doc []string) (scores []float64, inx int, strict bool, err error) {
+	if c.tfIdf && !c.didConvertTfIdf {
+		panic("Error:Using TD-IDF classifier:Need to call ConvertTermsFreqsToTFIDF's first ")
```

Comment:
Change error message to "Using a TF-IDF classifier. Please call ConvertTermsFreqToTfIdf before calling SafeProbScores."